app/helpers/ct_table_for/application_helper.rb

```diff
@@ -255,5 +256,11 @@ def uri?(string)
     def parse_extras(extras)
       Hash[extras.collect { |extra| [extra.split(":").first, extra.split(":").last] } ].with_indifferent_access
     end
+
+    def parse_custom_class(record, options = {})
+      style = options[:style].try(:call, record) || ""
+      tr_class = options[:tr_class] || ""
+      "class='#{tr_class} #{style}'"
```

Comment:
Please extract the attribute name to line 75. This method should only be responsible for generating the css styles.